dvc/repo/__init__.py

```diff
@@ -433,7 +432,9 @@ def _collect_stages(self):
         for root, dirs, files in self.tree.walk(self.root_dir):
             for file_name in filter(is_valid_filename, files):
                 path = os.path.join(root, file_name)
-                stages.extend(list(Dvcfile(self, path).stages.values()))
+                stage_loader = Dvcfile(self, path).stages
+                with stage_loader.log_level(at=logging.DEBUG):
+                    stages.extend(stage_loader.values())
```

Comment:
@efiop, one thing that changed here related to `pipeline file` is this ^.

When we collect all stages and there's no lockfile, we repeatedly get `No lock Entry` and is very noisy. But, again,this is very useful when loading single stage. So, I'm just keeping it to debug when collecting stages and set to WARNING on other times.